recipes/simpleitk/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "simpleitk" %}
+{% set version = "1.2.0" %}
+{% set sha256 = "7edac57f7a146163746b76e8bdeb9bf8b3cf3633c4e8e7ea3291121240d69d82" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SimpleITK/SimpleITK/releases/download/v{{version}}/SimpleITK-{{version}}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+    number: 0
+    skip: true  # [ win and py27 ]
+
+requirements:
+  build:
+    - cmake   >=3.11
+    - ninja
+    - swig    >=3
```

Comment:
probably should remove swig from here

Reply:
Without it being in both build and host, it was not able to locate the swig executable. Do you have a suggestion for setting SWIG_EXECUTABLE in the bat and sh files what would work with only a host requirement?

Reply:
swig is now only in the host, and BUILD_PREFIX is used to search for swig.